sed/core/processor.py

```diff
@@ -1321,6 +1327,90 @@ def calibrate_delay_axis(
             else:
                 print(self._dataframe)
 
+    def correct_delay_fluctuations(
+        self,
+        delay_column: str = None,
+        columns: Union[str, Sequence[str]] = None,
+        signs: Union[int, Sequence[int]] = None,
+        reductions: Union[str, Sequence[str]] = None,
+        preserve_mean: Union[bool, Sequence[bool]] = None,
+        **kwargs,
+    ) -> None:
+        """Apply a correction to the delay axis of the dataframe.
+
+        Args:
+            delay_column (str): Name of the column containing the delay values.
+            columns (Union[str, Sequence[str]]): Name of the column(s) to apply the correction to.
+            signs (Union[int, Sequence[int]]): Sign of the correction to apply. (+1 or -1)
+                A positive sign shifts the delay axis to higher delays. Defaults to +1.
+            preserve_mean (bool): Whether to subtract the mean of the column before applying the
+                correction. Defaults to False.
+            reductions (str): The reduction to apply to the column. Should be an available method
+                of dask.dataframe.Series. For example "mean". In this case the function is applied
+                to the column to generate a single value for the whole dataset. If None, the shift
+                is applied per-dataframe-row. Defaults to None. Currently only "mean" is supported.
+
+        Raises:
+            ValueError: If the delay column is not in the dataframe.
+        """
+        if delay_column is None:
+            delay_column = self._config["dataframe"]["delay_column"]
+        if delay_column not in self._dataframe.columns:
+            raise ValueError(
+                f"Delay column {delay_column} not found in dataframe! "
+                "Run `append delay axis` first.",
+            )
+        if self.dataframe is not None:
+            self._dataframe, metadata = self.dc.correct_delay_fluctuations(
+                df=self._dataframe,
+                delay_column=delay_column,
+                columns=columns,
+                signs=signs,
+                reductions=reductions,
+                preserve_mean=preserve_mean,
+                **kwargs,
+            )
+            self._attributes.add(
+                metadata,
+                "correct_delay_fluctuations",
+                duplicate_policy="raise",
+            )
+
+    def save_delay_calibration(
+        self,
+        filename: str = None,
+        overwrite: bool = False,
```

Comment:
That was something missing indeed, but there is also no loading for this for the mpes config yet. I'd suggest to change it similarly to the energy config as of now.